pyspectral/tests/test_raw_readers.py

```diff
@@ -23,15 +23,14 @@
 """Test the raw satellite instrument rsr readers."""
 
 import sys
-try:
-    from unittest import mock
-except ImportError:
-    import mock
-
 if sys.version_info < (2, 7):
     import unittest2 as unittest
 else:
     import unittest
+if sys.version_info < (3,):
+    import mock
+else:
+    from unittest import mock
```

Comment:
Why change the try/except to this? The previous version was doing it the "standard" way where python 3 is the expected behavior and python 2 is the exception.

Reply:
I was confused too! 
Previously I had `import mock` above the try-except. That doesn't work in python 3. It failed here at least....